### PR TITLE
Fix broken links

### LINKED ISSUE #89: Broken links over multiple places.
Broken links on the website.

### PR DESCRIPTION
**Pull request by nothuman2718**
Fix broken links as discussed in this [Closed PR](https://github.com/nightwatchjs/nightwatch-www/pull/81)
Closes #89

### CODE REVIEW

**Comment by nothuman2718** [comment]
Closes #89

File: src/includes/sections/about.ejs [hunk 1]
```diff
@@ -506,7 +506,7 @@
               </div>
               <div class="col p-4 d-flex flex-column position-static">
                 <strong class="d-inline-block mb-2">Article</strong>
-                <h3><a href="https://blog.percy.io/visual-testing-in-nightwatch-js-with-percy-1b68c122cf94"
+                <h3><a href="www.browserstack.com/guide/visual-regression-testing-in-nightwatchjs#toc3"
```
Comment (beatfactor) [inline src/includes/sections/about.ejs:509]: Missing https://

File: src/pages/guide/comparison-with-leading-frameworks.ejs [hunk 2]
```diff
@@ -30,12 +30,12 @@ const data = {
       <tr>
         <th></th>
         <th>
-          <img src="https://expr.nightwatchjs.org/images/symbol-nightwatch.svg" class="nightwatch-logo" />
+          <img src="https://expr.nightwatchjs.org/images/symbol-nightwatch.svg" class="nightwatch-logo" alt="nightwatch-logo" />
```
Comment (beatfactor) [inline src/pages/guide/comparison-with-leading-frameworks.ejs:33]: The alt attribute should be a descriptive text field

File: src/pages/guide/comparison-with-leading-frameworks.ejs [hunk 3]
```diff
@@ -141,7 +141,7 @@ const data = {
         <td>
           <span class="comp-yes">✅</span> <br>
           <p>Mobile web is just as easy to setup and supports everything from Desktop web.
-            <br><a href="#">Get Started</a>
+            <br><a href="ttps://nightwatchjs.org/guide/mobile-app-testing/introduction.html">Get Started</a>
```
Comment (beatfactor) [inline src/pages/guide/comparison-with-leading-frameworks.ejs:144]: Please check the href attribute.

File: src/pages/guide/comparison-with-leading-frameworks.ejs (hunk 2 above)
Comment (nothuman2718) [reply src/pages/guide/comparison-with-leading-frameworks.ejs]: Noted. Sure!!

File: src/pages/guide/comparison-with-leading-frameworks.ejs (hunk 3 above)
Comment (nothuman2718) [reply src/pages/guide/comparison-with-leading-frameworks.ejs]: Sorry, for underlooking and pushing these silly changes, Actually these changes are from my old PR. So, I simply copied from that PR, without testing them again. 
I will not repeat this again.

File: src/includes/sections/about.ejs (hunk 1 above)
Comment (nothuman2718) [reply src/includes/sections/about.ejs]: Sorry for underlooking and pushing these silly changes; actually, these changes are from my old PR. So, I copied from that PR without testing them again. 
I won't repeat this.

**Comment by nothuman2718** [comment]
@beatfactor requested changes are ready. Can you please look into this when you have time?

**Pull request merged**